src/chart/treemap/Breadcrumb.ts

```diff
@@ -185,7 +189,11 @@ class Breadcrumb {
                 onclick: curry(onSelect, itemNode)
             });
             (el as ECElement).disableLabelAnimation = true;
-
+            el.getTextContent().ensureState('emphasis').style = {text, ...createTextStyle(emphasisTextStyleModel)};
```

Comment:
`createTextStyle(emphasisTextStyleModel, { text })` is enough. No need to use spread operator.